api/src/queries/versions.js

```diff
@@ -46,7 +51,7 @@ const find = async (configurationId) => {
 const findOneByHash = async (configurationId, hash) => {
```

Comment:
This query didn't works because of the missing `hash` field in `searchableFields`.